contrib/face_recognition.lua

```diff
@@ -376,7 +387,7 @@ fc.known_image_path = dt.new_widget("file_chooser_button"){
   is_directory = true,
   changed_callback = function(this)
     dt.preferences.write(MODULE, "known_image_path", "directory", this.value)
-  end
+end
```

Comment:
please restore the indent since this closes the changed_callback function

Reply:
I used an online beautifier to make sure all my changes were correctly indented. It also changed some of your code below “-- build the interface”. I will rollback it.